src/site/content/en/design-system/welcome.njk

```diff
@@ -26,7 +26,7 @@ This will create the following folder and file structure:
 ```
 web.dev
 └── src/
-    └── pattern-library/
+    └── components/
```

Comment:
shouldn't this be cmponent-library?

Reply:
Component-library is the content url but makes sense to change the folder to that too tbh. Good catch

Reply:
lol ignore me it already was 😂